src/main/java/redlightBack/Board/BoardController.java

```diff
@@ -0,0 +1,39 @@
+package redlightBack.Board;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import redlightBack.Board.Dto.BoardCreateResponse;
+import redlightBack.Board.Dto.BoardRequest;
+import redlightBack.Board.Dto.BoardResponse;
+import redlightBack.loginUtils.LoginMemberId;
+
+import java.util.List;
+
+@RestController
+public class BoardController {
+
+    private final BoardService boardService;
+
+    public BoardController(BoardService boardService) {
+        this.boardService = boardService;
+    }
+
+
+    //게시판 생성
+    //관리자 권한 추가 필요
```

Comment:
참고로 이런 경우 아래와 같은 식으로 주석을 달아주셔도 좋습니다.
```suggestion
    // TODO: 관리자 권한 추가 필요
```